manifest/docker_schema1.go

```diff
@@ -95,14 +98,20 @@ func Schema1Clone(src *Schema1) *Schema1 {
 	return &copy
 }
 
-// initialize verifies invariants so that the rest of this code can assume a minimally healthy manifest.
+// initialize initializes ExtracedV1Compatibility and verifies invariants, so that the rest of this code can assume a minimally healthy manifest.
```

Comment:
Has an extra "initialize" in this comment.

Reply:
This is “$function $does_something”; maybe `s/initializes/prepares/` to be prettier?  I will fix the typo in `ExtracedV1Compatibility` at least.